Packs/Zafran/Integrations/Zafran/Zafran.py

```diff
@@ -0,0 +1,132 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+class Client(BaseClient):
+    def __init__(self, server_url, verify, proxy, headers, auth):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy, headers=headers, auth=auth)
+
+    def mitigation_performed_request(self,
+                                     mitigationstatus_external_ticket_id,
+                                     mitigationstatus_external_ticket_url,
+                                     mitigationstatus_id,
+                                     mitigationstatus_state):
+        data = assign_params(external_ticket_id=mitigationstatus_external_ticket_id,
+                             external_ticket_url=mitigationstatus_external_ticket_url, id=mitigationstatus_id,
+                             state=mitigationstatus_state)
+        headers = self._headers
+
+        response = self._http_request('post', 'mitigations/performed', json_data=data, headers=headers)
+
+        return response
+
+    def mitigations_export_request(self, filter_):
+        params = assign_params(filter=filter_)
+        headers = self._headers
+
+        response = self._http_request('get', 'mitigations', params=params, headers=headers)
+
+        return response
+
+    def mitigations_performed_request(self, mitigationsstatus_mitigation_id,
+                                      mitigationsstatus_mitigation_ids, mitigationsstatus_state):
+        data = assign_params(mitigation_id=mitigationsstatus_mitigation_id,
+                             mitigation_ids=mitigationsstatus_mitigation_ids, state=mitigationsstatus_state)
+        headers = self._headers
+
+        response = self._http_request('post', 'mitigations', json_data=data, headers=headers)
+
+        return response
+
+
+def mitigation_performed_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    mitigationstatus_external_ticket_id = str(args.get('mitigationstatus_external_ticket_id', ''))
```

Comment:
```suggestion
    mitigationstatus_external_ticket_id = args.get('mitigationstatus_external_ticket_id', '')
```
no need for cast it to str, the command arguments are strings. please change it in all the relevant places

Reply:
We are using the openapi codegen from the demisto-sdk. Will override manually.